code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -116,3 +116,18 @@
 	name = "moebius psychologist's turtleneck"
 	icon_state = "psychturtle"
 	item_state = "b_suit"
+
+/obj/item/clothing/under/rank/paramedic
+	desc = "Jumpsuit of Moebius Medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
+	name = "moebius paramedic's jumpsuit"
+	icon_state = "paramedic"
+	item_state = "paramedic"
+	permeability_coefficient = 0.50
+	armor = list(
+		melee = 10,
+		bullet = 5,
+		energy = 5,
+		bomb = 0,
+		bio = 30,
+		rad = 0
+	)
```

Comment:
Please add newline.

Reply:
what do you mean? Its the same format as all other uniforms?

Reply:
We'll need to run some kind of git script to go through and add newlines to all files.

Reply:
How does that even work? It's the first time i hear of this.

Reply:
The comment was directed to Reere, apologies, should of specifically mentioned them.